### PR TITLE
glpi

### PR DESCRIPTION
**Pull request by compgeniuses**
Adds GLPI App to NEthforge Repo: https://github.com/geniusdynamics/ns8-glpi

### CODE REVIEW

File: glpi/metadata.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  {
+    "name": "GLPI",
+    "description": {
+      "en": "GLPI is an incredible Service Management software based on open source technologies. It helps you plan and manage IT changes in an easy way, solve problems efficiently, automate your business processes and gain control over your IT infrastructure."
```
Comment (stephdl) [inline glpi/metadata.json:5]: please do not enhance a brand, explain in few words what aims the software

**Review by stephdl — CHANGES_REQUESTED** [review]

**Comment by stephdl** [comment]
We have shared together privately and your module have some defaults that you need to fix, this app seems needs some love on the initial configuration, the app asks for the database credential and maybe other issue, could you ask the test of the community to test this module before to ask to be included inside nethforge

**Comment by compgeniuses** [comment]
Updated the Metadata... Description

All Available Docker images that feature the latest release do require the user to enter database details on first login,
Not unless we build our own docker image to handle the specific case for Nethserver, We are forced to also implement it the same way, or workout a way ourselves on how to resolve the given challenge.

**Comment by stephdl** [comment]
could you try to mount a file `config_db.php` with this  content to the container where glpi files are 

adapt the user, password, db name to you need
```
<?php
class DB extends DBmysql {
   public $dbhost     = '127.0.0.1';
   public $dbuser     = 'glpi';
   public $dbpassword = '$db_pass';
   public $dbdefault  = 'glpi';
}
?>
```

normally glpi should be aware about the database credential

**Comment by compgeniuses** [comment]
Ok, thank you for your input

let me work out how to get this done and test.

**Comment by compgeniuses** [comment]
Hello, We opted to implementing our own docker here https://github.com/geniusdynamics/docker-glpi

we were able to get things past the database screen, however other challenges arose, still working on having the best possible way to achieve this

**Comment by compgeniuses** [comment]
hello @compgeniuses is there something pending with this App, coud you kindly restest and notify is anything is needed to be done

**Comment by stephdl** [comment]
released in genius repository

**Pull request closed without merging**